unfoldingWord/translationCore
# Fixed bug where target language was being replaced by a different projects target language when switching projects

#### This pull request addresses:

Fixed bug where target language was being replaced by a different projects target language when switching projects


#### How to test this pull request:

- Open a project with tN or tW
- Switch to a different project.
- Make sure The Resources/Bibles folder is not getting replaced by one from a different project.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1663)
